docs/custom-content.md

```diff
@@ -60,9 +60,6 @@ Peacock supports a broad variety of server side mods (plugins). These include, b
 -   Kill Everyone Mode
     -   All real NPC's marked as targets, can get Silent Assassin rating back after killing everyone.
     -   Note: Very buggy for maps with 200+ targets, guns may jam randomly. You may need to start by melee killing people.
--   Sarajevo Six
-    -   Bonus Missions otherwise only available in Hitman 2016 Playstation version.
-    -   Installation Instructions: https://github.com/solderq35/hitman-tech-tips/blob/main/modding/sarajevo_six.md
 -   Brothers Elusive Target (Cut Content)
     -   File Download: https://www.nexusmods.com/hitman3/mods/375?tab=files
     -   Install the main portion of the mod (the file not marked "Peacock Plugin") through Simple Mod Framework. Install the "Peacock Plugin" portion of the mod by putting it into your Peacock Install folder. Read the Sarajevo Six instructions above for guidance if confused about either of those steps.
```

Comment:
The Brother's section includes reference to "the Sarajevo Six instructions above". These no longer exist and this should be reworded. I also think it would be good to add a comment about this being a _fan interpretation_ and not how the mission originally played out.

Reply:
I removed that part from the Brother ET install instructions and also tried my best to clarify it is a fan-made mod / interpretation.